src/lotus_json/mod.rs

```diff
@@ -213,13 +214,17 @@ decl_and_test!(
     vrf_proof for crate::blocks::VRFProof,
 );
 
+mod beneficiary_term; // fil_actor_miner_state::v12::BeneficiaryTerm: !quickcheck::Arbitrary
```

Comment:
Do you find these comments helpful? I feel they can get outdated quickly, e.g., with the next actors' version bump, given there are no compile time checks and the truth is only in the module itself.

Reply:
Apart from the concern listed above I'm curious what actual purpose those serve. I'm assuming these are just comments carrying information about whether or not a certain type derives `Arbitrary`.

Reply:
I've made the purpose of these comments clearer

Reply:
wibni to `static_assertions::assert_not_impl_any` as a road bump, and I can if you feel strongly about this, but there is additional tech debt in that we're using `fil_actor_miner_state` _at all_ - I think these should be shim types. But this PR is only moving the impls, not fixing them

Reply:
We have done some wrapping on the fil-actor-states side for Claim and Allocation. Perhaps we could have something similar here, but not in this PR.